src/parameter_declarations.ts

```diff
@@ -143,7 +143,8 @@ Parameter.declare({
 		• parent folder ID (optional)
 			• --parent-folder-id used, otherwise
 			• all parent folder IDs should reside in the same drive
-		• Can NOT be used in conjunction with --local-file-path`
+		• Can NOT be used in conjunction with --local-file-path`,
+	forbiddenConjunctionParameters: [LocalFilePathParameter]
```

Comment:
We could generate the "Can NOT be used in conjunction with ..." or "Requires to be used in conjunction with ..." messages from the `forbiddenConjunctionParameters` and `requiredConjunctionParameters`